projects/igniteui-angular/src/lib/core/styles/components/grid/_grid-theme.scss

```diff
@@ -1958,6 +1958,11 @@
         z-index: 25;
     }
 
+    // Fix Grid column headers background has transparency on drag #10864
+    %grid__drag-ghost-image%igx-grid-th--selectable {
+        opacity: 1;
+    }
```

Comment:
Please nest this under the existing %grid__drag-ghost-image declaration.

Reply:
I added them under the %igx-grid-th--selectable since there is an "if" condition that I had to take into account.